### PR TITLE
IDE-2059 Clean App server does't work for ext project

### PR DESCRIPTION
**Pull request by lilu1990**

### CODE REVIEW

**Comment by lilu1990** [comment]
Greg ,

Do we need to remove the page to configure bundle zip in the "Edit Server Runtime Environment" page ?  Since now the bundle zip path is directly get from SDK properties file 

1 remove whole "next" page ( that include configure "Liferay Javadoc URL" , "Liferay source Location"  ,"Liferay Tomcat bundle Zip file" ) ?

2 only remove the "Liferay Tomcat bundle Zip file" part ?

**Comment by gamerson** [comment]
We can leave those settings on the tomcat/bundle for now.

File: tools/plugins/com.liferay.ide.server.tomcat.ui/src/com/liferay/ide/server/tomcat/ui/CleanAppServerAction.java [hunk 1]
```diff
@@ -199,11 +175,75 @@ protected void createChildFragments( List<WizardFragment> list )
...
+
+                boolean foundTomcat = false;
+
+                ZipEntry entry = zis.getNextEntry();
+
+                while( entry != null && !foundTomcat )
+                {
+                    String entryName = entry.getName();
+
+                    if( entryName.startsWith( rootEntryName + "/tomcat-" ) ) //$NON-NLS-1$
```
Comment (gamerson) [inline tools/plugins/com.liferay.ide.server.tomcat.ui/src/com/liferay/ide/server/tomcat/ui/CleanAppServerAction.java]: We should validate the zips with another path besides /tomcat

**Comment by lilu1990** [comment]
now clean app server can also works for jboss  , and  configure bundle zip part in "Edit Server Runtime Environment" page has been removed

**Comment by gamerson** [comment]
@lilu1990  can you please do option 2? Can you open a new ticket about this?  Thanks.

**Comment by gamerson** [comment]
I'll go ahead and merge this one.

**Comment by gamerson** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by gamerson** [comment]
Merged, thanks! See 8d23935...b3cda9a
8d23935...b3cda9a <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**